packages/cgb-scripts/scripts/eject.js

```diff
@@ -273,27 +274,25 @@ inquirer
 				],
 			],
 			plugins: [
-				[ 'transform-object-rest-spread' ],
+				'@babel/plugin-syntax-dynamic-import',
+				'@babel/plugin-syntax-import-meta',
 				[
-					'transform-object-rest-spread',
+					'@babel/plugin-proposal-class-properties',
```

Comment:
Why did you remove

- transform-object-rest-spread
- transform-object-rest-spread
- transform-react-jsx
- transform-runtime

🤔

Reply:
When running babel-upgrade, I expected the previous plugins to be included but it's possible that I am mistaken about that. 

I know that transform-react-jsx is included with 'preset-react' and I simply assumed that transform-runtime and transform-object-spread were similarly included in other ways:
https://babeljs.io/docs/en/babel-preset-react

In running this against my own guten blocks, everything compiled and is working correctly though it seems that more in depth testing may be needed.

Reply:
Have you test `-object-rest-spread` and it's working?